src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -137,25 +109,23 @@ export const TrackPanelBookmarks = () => {
   };
 
   return (
-    <section className="trackPanelBookmarks">
-      <div className={styles.title}>Bookmarks</div>
-      {exampleEnsObjects.length ? <ExampleLinks /> : null}
-      {limitedPreviouslyViewedObjects.length ? (
+    <section className={styles.trackPanelBookmarks}>
+      <div className={styles.title}>Previously viewed</div>
+      {previouslyViewedObjects.length ? (
         <>
-          <div data-test-id="previously viewed" className={styles.sectionTitle}>
-            Previously viewed
-            {previouslyViewedObjects.length > 20 && (
-              <span className={styles.ellipsis}>
+          <PreviouslyViewedLinks />
+          {previouslyViewedObjects.length > 20 && (
+            <div className={styles.more} onClick={onEllipsisClick}>
+              <span>more</span>
```

Comment:
I tried to see the "more" by switching between over 20 genes, but couldn't. Are you sure you are not removing previously viewed objects from the list if it gets longer than 20 items?

![image](https://user-images.githubusercontent.com/6834224/121611900-40028200-ca51-11eb-8347-c8ebc28e778a.png)

Reply:
Also, notice on the screenshot above that you are sometimes repeating stable ids twice (one unversioned, the other versioned). I believe the correct logic is to check if a gene has a symbol, and if it doesn't, then just display a versioned stable id.

Reply:
Yeah this was one of my doubt to Andrea.

Reply:
There is a bug...

Reply:
I can see the "more" link now; but the stable ids are still repeated if a gene does not have a symbol.

![image](https://user-images.githubusercontent.com/6834224/122481557-ccb7bd80-cfc6-11eb-9721-8434a669ff3f.png)

It might also be worth investigating whether there is anything that can/should be done to the styles of the previously viewed links in the drawer. Currently we crop long lines, which may look weird:

![image](https://user-images.githubusercontent.com/6834224/122482222-0b01ac80-cfc8-11eb-9ba7-da70bba0ec04.png)

I understand that there are several CSS requirements at once that we are trying to balance here (the number of the columns, etc; so there may not be a good CSS solution for the problem that we have).

Finally, notice that there are two vertical scroll bars in the sidebar. This is probably due to how the modal is implemented. Ideally, we should only have a single scrollbar in the sidebar area.